docs/README-develop-node-utils-APIs.md

```diff
@@ -82,6 +84,12 @@ git clone https://github.com/YOUR-USERNAME/cisco-network-node-utils.git
 cd cisco-network-node-utils/
 ```
 
+If you're using Bundler, invoke it to ensure all of the development dependencies are installed on your system:
+
+```bash
+bundle install
```

Comment:
Seems to, assuming you've installed Bundler using --local-install and have your PATH set correctly - though it prompts you in that case to specify a non-default install location, that works as documented:

```
> ~/.gem/ruby/2.1.0/bin/bundle install
Fetching gem metadata from https://rubygems.org/.........
Fetching version metadata from https://rubygems.org/..
Resolving dependencies...
Using rake 10.4.2
Using ast 2.0.0
Using parser 2.3.0.pre.2
Using astrolabe 1.3.0
Using bundler 1.10.4
Using net_http_unix 0.2.1


Your user account isn't allowed to install to the system Rubygems.
You can cancel this installation and run:

    bundle install --path vendor/bundle

to install the gems into ./vendor/bundle/, or you can enter your password
and install the bundled gems to Rubygems using sudo.

Password:^C
```

```
> ~/.gem/ruby/2.1.0/bin/bundle install --path vendor/bundle
Fetching gem metadata from https://rubygems.org/.........
Fetching version metadata from https://rubygems.org/..
Resolving dependencies...
Installing rake 10.4.2
Installing ast 2.0.0
Installing parser 2.3.0.pre.2
Installing astrolabe 1.3.0
Using bundler 1.10.4
Installing net_http_unix 0.2.1
Installing cisco_nxapi 1.0.0
Using cisco_node_utils 1.0.2.pre.dev from source at .
Using minitest 4.7.5
Installing powerpack 0.1.1
Installing rainbow 2.0.0
Installing ruby-progressbar 1.7.5
Installing rubocop 0.34.2
Bundle complete! 5 Gemfile dependencies, 13 gems now installed.
Bundled gems are installed into ./vendor/bundle.
```